src/Main.java

```diff
@@ -1,5 +1,57 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        System.out.println("Задача 1");
+        int [] expenses = {500, 2000, 300, 1500, 800};
+        int sum = 0;
+        int sum1 = 0;
+
+        for (int i = 0; i < expenses.length; i++) {
+            sum += expenses[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+
+        for (int element : expenses) {
+            sum1 += element;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum1 + " рублей");
+        System.out.println();
+
+        System.out.println("Задача 2");
+        int maxExpenses = -1;
+        int minExpenses = 2500;
```

Comment:
Если минимальная сумма трат будет 3000, то твой код перестанет работать. Чтобы этого избегать обычно изначальное значение приравнивают к значению первого элемента массива
```suggestion
        int minExpenses = expenses[0];
```